helm/hpcc/values.yaml

```diff
@@ -378,6 +377,10 @@ roxie:
   #serverResources:
   #  cpu: "1"
   #  memory: "1G"
+  topoServer:
+    replicas: 1
+    traceLevel: 1
+    port: 9003
```

Comment:
in the defaultYaml in the cpp, port is 9004.. probably should be same here?

Reply:
Oops, yes, left in for testing. I will fix